src/js/custom_shields.js

```diff
@@ -0,0 +1,73 @@
+"use strict";
+
+import * as Color from "../constants/color.js";
+import * as ShieldDraw from "./shield_canvas_draw.js";
+
+// Special case for Allegheny, PA Belt System, documented in CONTRIBUTE.md
+export function paBelt(params) {
+  let ctx = ShieldDraw.roundedRectangle({
+    fillColor: Color.shields.white,
+    strokeColor: Color.shields.black,
+    outlineWidth: 1,
+    radius: 2,
+    rectWidth: 20,
+  });
+
+  let fillColor = params.fillColor;
+
+  let lineWidth = 0.5 * ShieldDraw.PXR;
+  let diameter = ShieldDraw.CS / 3 - lineWidth;
+  ctx.beginPath();
+  ctx.arc(
+    ShieldDraw.CS / 2,
+    ShieldDraw.CS / 2,
+    diameter,
+    0,
+    2 * Math.PI,
+    false
+  );
+
+  ctx.fillStyle = fillColor;
+  ctx.strokeStyle = Color.shields.black;
```

Comment:
How come `strokeColor` is no longer a parameter and this is set to black?

Reply:
Are you sure? I see white used as an outline for the [green, blue and purple belts](https://github.com/ZeLonewolf/openstreetmap-americana/pull/708/files#diff-21d2ca32bb400fce6149c2ca25ed3366df195a198f357764d46f7ae811835f29L4158-L4160).

Reply:
Interesting, I'm thinking that was actually a bug though?  Shouldn't they all be black bordered?

![image](https://user-images.githubusercontent.com/3254090/212972270-24ac960f-1dd7-43d5-a1d4-3442af4a1435.png)

Reply:
It was a stylistic choice to improve color contrast: https://github.com/ZeLonewolf/openstreetmap-americana/pull/356#issuecomment-1144137548

Same thing with the Branson routes. We could revisit this decision, but that'd be appropriate for a separate PR.

Reply:
Perfect, thanks for the dive on that.  I'll edit this to preserve the current behavior.